book/src/proposals/broadmoor.md

```diff
@@ -215,10 +214,9 @@ blockquote {
 </style>
 
 You could be forgiven for not knowing much about the Broadmoor neighborhood in
-Seattle &mdash; that's kind of by design. Using the
-[A / B Street](../software/abstreet.html) travel simulation game, we're going to
-explore how it could become a valuable link for people walking and biking in
-Seattle.
+Seattle &mdash; that's kind of by design. We're going to explore how it could
+become a valuable link for people walking and biking in Seattle, using the
+travel simulation game [A/B Street](../software/abstreet.html).
```

Comment:
Upping the pressure on me to finish this page! This is fine. Pressure's good. :)